src/components/calendar/index.tsx

```diff
@@ -48,15 +50,19 @@ export class Calendar {
 							<td
 								tabindex={1}
 								onClick={() => {
-									this.valueChanged.emit((this.value = date))
-									this.clickCounter += 1
-									if (this.clickCounter % 2 == 1)
-										this.start = this.end = date
-									else {
-										if (this.start && date > this.start)
-											this.end = date
-										else
-											this.start = date
+									if (
```

Comment:
All buttons are getting an onclick function added to them now, even if they are disabled.
I suggest:
``` tsx
onClick={!(Date.parse(date) <= Date.parse(this.min ?? "") || Date.parse(date) >= Date.parse(this.max ?? "")) 
? () => onClick(date) 
: undefined
```
And then move out the logic in the if statement into a function `private onClick(date: Date)`